WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -107,15 +107,15 @@ struct CheckmarkRow: ImmuTableRow {
 }
 
 struct LinkRow: ImmuTableRow {
-    static let cell = ImmuTableCell.class(WPTableViewCellValue1.self)
+    static let cell = ImmuTableCell.class(WPExternalLinkCell.self)
```

Comment:
I'm not sure you need a custom subclass. Unless I missed some details, you could use a `WPTableViewCellDefault` and create/set the `accessoryView` in `configureCell(_:)`

Reply:
You're right! I think I initially wrote it when I thought I'll add `ExternalLinkRow` in addition to `LinkRow`, but it wasn't needed in the first place at all.